src/GitLink/Pdb/SrcSrvContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="SrcSrvContext.cs" company="CatenaLogic">
+//   Copyright (c) 2014 - 2016 CatenaLogic. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace GitLink.Pdb
+{
+    using System.Collections.Generic;
+
+    public class SrcSrvContext
```

Comment:
I'm not a C# coder by trade, so let me know whether or not I'm using this class properly...